include/openthread/link.h

```diff
@@ -1182,6 +1182,21 @@ bool otLinkIsCslSupported(otInstance *aInstance);
  */
 otError otLinkSendEmptyData(otInstance *aInstance);
 
+/**
+ * Sets the region code.
+ *
+ * The radio region format is the 2-bytes ascii representation of the ISO 3166 alpha-2 code.
```

Comment:
Just curios: Does the endian-ness of `uint16_t` value matter here? I guess probably no, is this correct?
I guess we always assume the first ascii char is `aRegionCode >> 8` and second char is `aRegionCode & 0xff`?

Reply:
Good question. The bit-shift doesn't depend on endian-ness, see discussion at   https://stackoverflow.com/questions/7184789/does-bit-shift-depend-on-endianness.  So the endian-ness of `uint16_t` value doesn't matter here.

Reply:
Sounds good. 

We should make sure the platform implementation don't try to cast and parse it as a `char region[2]` (since then it will depend on endian-ness).

Reply:
I updated the comments of the function `SetRegion()` to explain the format of the `RegionCode`. That should help the platform implementation to better understand the format.